metatron-app/metatron-discovery
# #4048 Adds a tooltip that is always visible on the map chart.

### Description

Adds a tooltip that is always visible on the map chart.

**Related Issue** : #4048 


### How Has This Been Tested?

1. Create a map chart.
2. Set the marker in the tooltip label setting.
3. Check that the tooltips that are always displayed in the map chart are normally displayed.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
